openpdf/src/main/java/com/lowagie/text/pdf/PdfDocument.java

```diff
@@ -776,6 +770,23 @@ public boolean add(Element element) throws DocumentException {
         }
     }
 
+    static PdfPTable createWithAllElementsInOneCell(Paragraph paragraph) {
```

Comment:
It's a bit confusing to name method `WithAllElements` while passing single `Paragraph` object without looking into the code. I would consider renaming to smth like `createIn(Single|One)Cell` or pass `Iterator` instead of `Paragraph` (so it may be reused)